pennylane_lightning/core/src/simulators/lightning_tensor/utils/cutn_utils/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.20)
```

Comment:
Why is this under the `cutn_utils` directory? Since `utils` is not so crowded, can't we just put stuff in there?

Reply:
yea, we can put put everything into the `utils`. I put them into `cutn_utils` since those files are exclusively for `cutensornet` support. This could also make it easier if we want to add supports to other backends.